integration_tests/steps/integration_tests.py

```diff
@@ -1064,3 +1072,20 @@ def identical_profiles(context):
     start = int(list(load_profile_ts.keys())[0])
     end = int(list(load_profile_ts.keys())[23])
     assert all([load_profile_ts[i] == load_profile_ts[i + 86400] for i in range(start, end, 3600)])
+
+
+def get_simulation_raw_results(context):
+    area_tree_summary = glob.glob(os.path.join(context.export_path, "*", "area_tree_summary.json"))
+    print(f"area_tree_summary: {area_tree_summary}")
+    with open(area_tree_summary[0], "r") as sf:
+        context.area_tree_summary_data = json.load(sf)
+    from d3a.d3a_core.util import area_name_uuid_map
```

Comment:
I assume that you have tested this, and it is required to be imported lazily. I am asking because it would be great if we could avoid lazy import here.

Reply:
no, just double checked, moving it to top